swagger.json

```diff
@@ -2206,8 +2444,80 @@
           }
         }
       },
+      "put": {
```

Comment:
Replacing a job. The process graph might be needed to be changed without creating a new job id. This sounds especially useful as we removed the DELETE endpoint for jobs. Otherwise every change would trigger a new URL for (shared) jobs. We still need to think about what happens with running jobs when patching/changing. etc.